### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by flean** [maximummeteor/server-transform#3]
I add your package and get this error

```
E20150929-11:04:15.083(-7) (webapp_server.js:701) Error running template: TypeError: Object [object Object] has no method 'onStop'
```

**Comment by maxnowack** [comment]
Thank you for reporting this issue!
I'll try to fix this bug in the next few days

**Comment by crapthings** [comment]
+1 i have same problem

**Comment by vraptor75011** [comment]
+1

**Comment by maxnowack** [comment]
should be fixed in v0.4.1

**Comment by crapthings** [comment]
awesome, thanks @maxnowack
